app/controllers/items_controller.rb

```diff
@@ -1,20 +1,21 @@
 class ItemsController < ApplicationController
-  def home
-    @item = Item.new
-    @items = Item.all
-  end
 
   def create
-    @item = Item.new(item_params)
+    @inventory_dashboard = InventoryDashboard.new(Item.all)
 
-    if @item.save
-      redirect_to root_path
+    if @inventory_dashboard.save_item(item_params)
+      flash[:error] = 'error message'
+      render :index
     else
-      flash[:error] = "Invalid"
+      flash[:success] = 'success'
       redirect_to root_path
     end
   end
 
+  def index
+    @inventory_dashboard = InventoryDashboard.new(Item.all)
+  end
+
```

Comment:
@BrandonMathis - when you have a second:

I created the InventoryDashboard class in lib and configured Rails to auto load my lib files. My unit tests in spec/lib for this file are all passing. But I get an error when I try to render a view that @item, @category, etc variables being used in my view are nil or empty:

ArgumentError in Items#index: First argument in form cannot contain nil or be empty. See screenshot. I get the same error when I try to render the category_form view or the index view without either form. 

<img width="1026" alt="screen shot 2016-01-21 at 2 03 04 pm" src="https://cloud.githubusercontent.com/assets/12813762/12491191/b9ac9af6-c047-11e5-8fcf-7f2b407132c9.png">

I thought that initializing the dashboard would create the other variables so I could use them in views. Any guidance here?

(P.S. I'm about to head out until Tuesday but I'm taking my computer! Have a great weekend)